### PR TITLE
Added Portuguese-Brazil strings

### PR DESCRIPTION
**Pull request by Bruno-Furtado**

### CODE REVIEW

File: Airbrake-iOS.podspec [hunk 1]
```diff
@@ -17,7 +17,7 @@ Pod::Spec.new do |s|
   
   s.platform     = :ios, "6.0"
 
-  s.source       = { :git => "https://github.com/airbrake/airbrake-ios.git", :tag => "4.2.1" }
+  s.source       = { :git => "https://github.com/tdginternet/airbrake-ios.git", :tag => "4.2.1" }
```
Comment (jocelynlih) [inline Airbrake-iOS.podspec]: please revert this change then we can merge this PR

**Comment by Bruno-Furtado** [comment]
@jocelynlih,
done.
